src/assets/server/rest/v1/schemas/tenant/tenant-data-update.json

```diff
@@ -10,13 +10,41 @@
       "$ref": "tenant#/definitions/name"
     },
     "email": {
-      "$ref": "user#/definitions/email"
+      "$ref": "common#/definitions/email"
     },
     "logo": {
       "$ref": "common#/definitions/logo"
     },
     "address": {
       "$ref": "common#/definitions/address"
+    },
+    "support": {
+      "$ref": "common#/definitions/tenantSupport"
+    },
+    "corporateName": {
+      "type": "string",
+      "sanitize": "mongo"
+    },
+    "website": {
+      "type": "string",
+      "sanitize": "mongo"
+    },
+    "rcs": {
+      "type": "string",
+      "sanitize": "mongo"
+    },
+    "capital": {
+      "type": "number",
+      "sanitize": "mongo"
+    },
+    "siret": {
+      "type": "number",
+      "sanitize": "mongo",
+      "pattern": "^[0-9]{14}$"
+    },
+    "legalStatus": {
+      "type": "string",
+      "sanitize": "mongo"
     }
```

Comment:
Maybe all this data goes to common.js as it can be reused in Company ;P
And why not making it a component like the address ?

Reply:
it could be reused by companies but we won't use it as seen with @ClaudeROSSI

Reply:
and yes i should have made a new component as address or support 🤷🏻‍♀️